vagrant_variables.yml

```diff
@@ -17,4 +17,4 @@ memory: 8192
 #   - https://atlas.hashicorp.com/boxes/search?utf8=✓&sort=&provider=virtualbox&q=
 #   - https://download.gluster.org/pub/gluster/purpleidea/vagrant/
 vagrant_box: contiv/centos72
-box_version: 0.3.2
+box_version: 0.6.0
```

Comment:
Yep

On 14 May 2016, at 5:04, unclejack wrote:

>> @@ -17,4 +17,4 @@ memory: 8192
>>  #   - 
>> https://atlas.hashicorp.com/boxes/search?utf8=✓&sort=&provider=virtualbox&q=
>>  #   - https://download.gluster.org/pub/gluster/purpleidea/vagrant/
>>  vagrant_box: contiv/centos72
>> -box_version: 0.3.2
>> +box_version: 0.6.0
>
> Do we have both VirtualBox and VMware boxes for this version?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/256/files/a08a71ddffadebfe2bf11609facbd547c17a4f92..c7466bbb368cc39c2a41c34fa1452c2843e7dfcd#r63275202